packages/scheduler/src/components/FullCalendar/FullCalendar.js

```diff
@@ -114,16 +114,19 @@ class FullCalendar extends React.Component {
     this.forceUpdate()
   }
 
-  componentWillReceiveProps(newProps) {
-    this.calendarElement.fullCalendar('option', this.getFullCalendarOptions(newProps))
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (!_isEqual(prevProps, this.props)) {
```

Comment:
does it really need a full compare of props with prevProps? getFullCalendarOptions does not use the whole prop object. Maybe you could write a more specific if